assignments/lambda-classes.js

```diff
@@ -1 +1,115 @@
-// CODE here for your Lambda Classes
+//base class
+
+class Person {
+  constructor(props) {
+    this.name = props.name;
+    this.age = props.age;
+    this.location = props.location;
+    this.gender = props.gender;
+    this.catchPhrase = props.catchPhrase;
+  }
+  speak(){
+    return `Hello my name is ${this.name}, I am from ${this.location}`
+  }
+}
+
+class Instructor extends Person {
+  constructor(instructorProps) {
+    super(instructorProps);
+    this.specialty = instructorProps.specialty;
+    this.favLanguage = instructorProps.favLanguage;
+    }
+    demo(subject){
+    return  `Today we are learning about ${subject}`
+    }
+    grade(student, subject){
+      return `${student.name} receives a perfect score on ${subject}`
+    }
+}
+
+class Student extends Person {
+  constructor(studentParams) {
+    super(studentParams);
+    this.previousBackground = studentParams.previousBackground;
+    this.className = studentParams.className;
+    this.favSubjects = studentParams.favSubjects;
+  }
+
+  listsSubjects(skill1, skill2, skill3){
```

Comment:
This method needs no parameters, as we already have this information on the object in the `favSubjects` array. All you need to do here is loop through that array, and log each subject.  Try making this improvement, and let me know when you resolve it.